lib/RestructureCFG/InlineDivergentScopesPass.cpp

```diff
@@ -358,6 +401,18 @@ performMultipleIDS(ScopeGraphBuilder &SGBuilder,
     MultipleDivergences.erase(MultipleDivergences.begin() + LastElementIndex);
   }
 
+  // Decide if we need a new `Head` that chokes the entry of the divergent
```

Comment:
Perhaps "collects" is better than "choke". You've used this both in comments and in the commit message. Please fix.